gremlin-dotnet/src/pom.xml

```diff
@@ -65,14 +65,16 @@ limitations under the License.
                             </properties>
                             <scripts>
                                 <script>
-def file = new File(projectBaseDir,"Gremlin.Net/Gremlin.Net.csproj")
-file.write(file.getText("UTF-8").replaceFirst(/&lt;Version&gt;(.*)&lt;\/Version&gt;/, "&lt;Version&gt;" + mavenVersion + "&lt;/Version&gt;"))
+                                    def mavenVersion = "${project.version}"
```

Comment:
`mavenVersion` is already defined as a property above in line 62 here so I guess this shouldn't be needed?

Reply:
addressed in [553b146](https://github.com/apache/tinkerpop/pull/1717/commits/553b14630a5fe341cbb27cf06a65e4f53e828d2f)